noxfile.py

```diff
@@ -0,0 +1,88 @@
+"""
+Copyright 2019 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+
+from __future__ import absolute_import
+import os
+import shutil
+import sys
+import nox
+
+BLACK_PATHS = ["connector.py", "tests", "noxfile.py"]
```

Comment:
It may make sense to use patterns. For instance `*.py` to get all in root. If that isn't desirable though by name is good.